.pre-commit-config.yaml

```diff
@@ -25,19 +25,14 @@ repos:
   rev: 7.1.1
   hooks:
   - id: flake8
-    additional_dependencies: [
-      'flake8-annotations==3.0.1'
-    ]
     args: [
       --exclude, src/ansys/fluent/core/generated,
-      --select, W191 W291 W293 W391 E115 E117 E122 E124 E125 E225 E231 E301 E303 F401 F403 N801 N802 N803 N804 N805 N806,
```

Comment:
Removing this selection set selects this set+ the other default checks from flake8 except those in `--extend-ignore` below.